rebasebot/bot.py

```diff
@@ -426,6 +426,15 @@ def _init_working_dir(
     return gitwd
 
 
+def _manual_rebase_in_repo(rebase_repo) -> bool:
```

Comment:
just `repo` would be better i think.

If possible, can you add a type hint for this?